CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- `SetOrderPayment` mutation to update payment data
```

Comment:
can you remove these empty spaces around the titles? we found out it makes rebasing more complicated with unnecessary conflicts in the changelog. you can remove the spaces of the `0.1.0` release as well
```suggestion
## [Unreleased]
### Added
- `SetOrderPayment` mutation to update payment data
```

Reply:
It makes sense 💯